src/main/java/hw_3/Task1.java

```diff
@@ -0,0 +1,33 @@
+package hw_3;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+import static java.awt.SystemColor.text;
+public class Task1 {
+    public static void main(String[] args) {
+        // ДЗ рядки
+        //Виведіть у консоль один рядок з числами з масиву (які ми створювали в 1-му завданні), розділеними “;”
+        int[] array = {1, 2, 3, 4, 5};
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < array.length; i++) {
+            result.append(array[i]);
+            if (i < array.length - 1) {
+                result.append(";");
+            }
+        }
+        System.out.println(result.toString());
+        // Створіть змінну із рядком “Мама мила раму”.
+        //Створіть масив зі всіх слів, які зустрічаються у цьому рядку.
+        String message = "Mother washed the frame";
+        String[] words = message.split(" ");
+        for (String word : words) {
+            System.out.println(word);
+        }
+        /*Створіть рядок "Я хочу перенесення рядків."
+        Замініть всі символи пробілу в рядку на символи перенесення рядків та виведіть у консоль.*/
+        String str = "I want line breaks";
+        String replacedstr = str.replace(" ", "/");
```

Comment:
"/" is the wrong symbol. You should to use
![Screenshot_12](https://github.com/AntBelyi/ALevel_automation/assets/66726924/5bed985b-d609-4040-883a-43113541de04)

So, all words in console should be from the new line

Reply:
fixed

Reply:
not